scripts/upload-iso

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# This script is used to upload Qubes OS release ISO.
+
+set -e
+[ "$DEBUG" = "1" ] && set -x
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 ISO_NAME"
+    echo " ISO_NAME is iso filename"
+    exit 1
+fi
+
+ISO="$(readlink -f "$1")"
+ISO_NAME="$(basename "$ISO")"
+ISO_BASE="${ISO_NAME%%.iso}"
+
+[ "x$HOST" == "x" ] && HOST=yum.qubes-os.org
+[ "x$HOST_ISO_BASEDIR" == "x" ] && HOST_ISO_BASEDIR=/pub/qubes/iso
+
+echo "Uploading $ISO..."
+rsync -av --progress "$ISO" "$ISO.asc" "$ISO.DIGESTS" "$HOST:$HOST_ISO_BASEDIR/$ISO_BASE$ISO_SUFFIX.iso"
```

Comment:
This won't work. It will put all the files into a new _directory_ named `$ISO_BASE$ISO_SUFFIX.iso`. I don't think rsync can rename multiple files on the fly. You either need separate rsync calls, or have target names locally. You can use a trick similar to create-torrent script - create temporary directory, hardlink files there under desired names, rsync it and remove.

Reply:
It's a mistake indeed.